include/bm/grpc/pem.h

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2013-present Barefoot Networks, Inc.
+ * Copyright 2022 VMware, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef BM_GRPC__PEM_H_
```

Comment:
The code style checker doesn't run in this directory apparently. If it did, the check would fail because of the extra underscore in the include guard.

Reply:
Thank you for pointing this out. I've updated the check_style.sh script to run on this file as well and fixed the code style warnings.